frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java

```diff
@@ -71,11 +67,6 @@ default void syncToRemoteStatusDependentActionUpdate(AnActionEvent anActionEvent
     var isRelationEligible = getEligibleRelations().contains(relation);
 
     if (isRelationEligible) {
-      if (getCurrentBranchNameIfManaged(anActionEvent).contains(branchName)) {
-        presentation.setText(
-            format(getString("action.GitMachete.ISyncToRemoteStatusDependentAction.text.current-branch"), getActionName()));
-      }
-
       var enabledDesc = format(getEnabledDescriptionFormat(), getActionNameForDescription(), branchName);
```

Comment:
Whoops... `branchName` is an `Option` :/

Reply:
Actually, I'm fixing this and related issues on a separate PR, no action required here

Reply:
Whoops... My fault. Corrected as in `ISyncTOParentStatusDependentAction`